libs/config/src/appConfig/cluster.ts

```diff
@@ -29,9 +29,73 @@ export type ClusterConfigSchema = Static<typeof ClusterConfigSchema>;
 
 const CLUSTER_CONFIG_BASE_PATH = "clusters";
 
+
+//将partitions内容改为列表
+export const ClusterListConfigSchema = Type.Object({
+  displayName: Type.String({ description: "集群的显示名称" }),
+  scheduler: Type.Enum({ slurm: "slurm" }, { description: "集群所使用的调度器，目前只支持slurm", default: "slurm" }),
+  slurm: Type.Object({
+    loginNodes: Type.Array(Type.String(), {  description: "集群的登录节点地址", default: []}),
+    computeNodes: Type.Array(Type.String(), { description: "集群的计算节点地址", default: []}),
+    partitions: Type.Array(
+      Type.Object(
+        {
+          name: Type.String({ description: "分区名" }),
+          mem: Type.Integer({ description: "内存，单位M" }),
+          cores: Type.Integer({ description: "核心数" }),
+          gpus: Type.Integer({ description: "GPU卡数" }),
+          nodes: Type.Integer({ description: "节点数" }),
+          qos: Type.Optional(Type.Array(Type.String({ description: "QOS" }))),
+          comment: Type.Optional(Type.String({ description: "计费项说明" })),
+        }
+      ),
+      {
+        description: "分区信息，分区名、内存、核心数、GPU卡数、节点数、QOS、计费项说明",
+        default: []
+      }
+    ),
+    mis: Type.Optional(SlurmMisConfigSchema),
+  }),
+  misIgnore: Type.Boolean({ description: "在实际进行MIS操作时忽略这个集群", default: false }),
+});
+
+export type ClusterListConfigSchema = Static<typeof ClusterListConfigSchema>;
+
+const List2Map = (src: Record<string, ClusterListConfigSchema>): Record<string, ClusterConfigSchema> => {
```

Comment:
Naming convention, and use a full sentence. `List2Map` -> `convertToOldConfigSchema`